docs/getting-started/products/yvaults/v3.md

```diff
@@ -0,0 +1,244 @@
+- `DEBT_MANAGER`: role for accounts that can rebalance and manage debt-related parameters.
+- `EMERGENCY_MANAGER`: role for accounts that can activate the shutdown mode.
+- `ACCOUNTING_MANAGER`: role for accounts that can process reports (save strategy p&l) and change accountant parameters.
+
+Every role can be filled by an externally owned account, multi-signature wallet, or other smart contracts. Each role can be filled by several accounts.
+
+The account that manages roles is a single account, set in `role_manager`.
+
+This `role_manager` can be an externally owned account, a multi-signature wallet, or a Governance Module that relays calls. 
+
+## Strategy Minimum API
+Strategies are completely independent smart contracts that can be implemented following the proposed template or in any other way.
+
+In any case, to be compatible with the vault, they need to implement the following functions, which are a subset of ERC4626 vaults: 
+- `asset()`: View returning underlying asset.
+- `vault()`: View returning vault this strategy is plugged to.
+- `totalAssets()`: View returning current amount of assets. It can include rewards valued in `asset`.
+- `maxDeposit(address)`: View returning the amount max that the strategy can take safely.
+- `deposit(assets, receiver)`: Deposits `assets` amount of tokens into the strategy. It can be restricted to the vault only or be open.
+- `maxWithdraw(address)`: View returning how many assets can the vault take from the vault at any given point in time.
+- `withdraw(assets, receiver, owner)`: Withdraws `assets` amount of tokens from the strategy.
+- `balanceOf(address)`: Return the number of shares of the strategy that the address has.
+
+This means that the vault can deposit into any ERC4626 vault and that a non-compliant strategy can be implemented provided that these functions have been implemented (even in a non ERC4626 compliant way). 
+
+Anything else is left to the strategy writer. However, to make security review easier, the Yearn's template has the following optional functions: 
+- `tend()`: a function that will be called by bots. It will do anything to maintain a position, such as act on certain triggers.
+- `tendTrigger()`: implementation that trigger bots that will call tend function on the contract.
+- `invest()`: deposit funds into underlying protocol.
+- `emergencyFreeFunds()`: close the position and return funds to the strategy. Losses might be accepted here.
+
+
+## ERC4626 compliance
+Vault Shares are ERC4626 compliant. 
+
+The most important implication is that `withdraw` and `redeem` functions as presented in ERC4626, the liquidity to redeem shares will be the one in the vault. No strategies will be passed to the redeem function to withdraw from with the ERC4626 compliant `withdraw` and `redeem` function. 
+
+## Emergency Operation
+
+### Shutdown mode
+
+If the current roles stop fulfilling their responsibilities or something else happens, the `EMERGENCY_MANAGER` can shut down the vault.
```

Comment:
`something else happens` > `an error occurs` might sound better?

Reply:
used jmonteers wording to update this sentence